modules/widgets/facebook-likebox.php

```diff
@@ -207,17 +211,36 @@ function form( $instance ) {
 			</label>
 		</p>
 
+		<p>
+			<label for="<?php echo esc_attr( $this->get_field_id( 'hide_cta' ) ); ?>">
+				<input type="checkbox" name="<?php echo esc_attr( $this->get_field_name( 'hide_cta' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'hide_cta' ) ); ?>" <?php checked( $like_args['hide_cta'] ); ?> />
+				<?php _e( 'Hide Call To Action Button', 'jetpack' ); ?>
+				<br />
+				<small><?php _e( 'Hides the custom call to action button (if available).', 'jetpack' ); ?></small>
+			</label>
+		</p>
+
+		<p>
+			<label for="<?php echo esc_attr( $this->get_field_id( 'small_header' ) ); ?>">
+				<input type="checkbox" name="<?php echo esc_attr( $this->get_field_name( 'small_header' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'small_header' ) ); ?>" <?php checked( $like_args['small_header'] ); ?> />
+				<?php _e( 'Use Small Header', 'jetpack' ); ?>
+				<br />
+			</label>
+		</p>
+
 		<?php
 	}
 
 	function get_default_args() {
 		$defaults = array(
-			'href'       => '',
-			'width'      => $this->default_width,
-			'height'     => $this->default_height,
-			'show_faces' => 'true',
-			'stream'     => '',
-			'cover'      => 'true',
+			'href'       		=> '',
+			'width'      		=> $this->default_width,
+			'height'     		=> $this->default_height,
+			'show_faces' 		=> 'true',
+			'stream'     		=> '',
+			'cover'      		=> 'true',
+			'hide_cta'      	=> '',
+			'small_header'      => '',
```

Comment:
Need to fix issues with spacing to make it better formatted. Will do that after initial review by someone! :)